### PR TITLE
GRD-617

### PR DESCRIPTION
**Pull request by hannahdriver**
Updating README to include commands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,62 @@ Output | Type | Description
...
+             for (i in counts) {
+                     print counts[i] "," i
+             }
+     }' | \
+     sort -nr | \
+     gzip -n > "~{outputFileNamePrefix}counts.gz"
+ ``` 
+
+
 ## Niassa + Cromwell
```
Comment (pruzanov) [inline README.md]: THis section is not needed since we are not using Niassa anymore. It is a relic from the past

**Comment by bibilujan** [comment]
@hannahdriver can you add a CHANGELOG 
Unreleased #date
#your changes

## 1.0.2 - 2021-05-28
- migrate to Vidarr

File: README.md (hunk 1 above)
Comment (bibilujan) [inline README.md]: Can you remove this section of the README, its not needed anymore as Niassa no longer exists.

**Comment by hannahdriver** [comment]
> @hannahdriver can you add a CHANGELOG Unreleased #date #your changes
> 
> ## 1.0.2 - 2021-05-28
> * migrate to Vidarr

Yes, done!

**Review by pruzanov — CHANGES_REQUESTED** [review]
Also, I believe it would be better if you consider prettifying the command section - making it more readable, see delly for an example

File: README.md (hunk 1 above)
Comment (hannahdriver) [reply README.md]: Deleted, thanks!

Sorry, I'm not sure I understand what you mean by prettifying the command section. I noticed in the delly README, the headers for each command aren't bolded - would you like me to change the headings to normal text for this one?

Comment (hannahdriver) [reply README.md]: Deleted, thanks!

I have updated the commands section for readability - let me know if it looks okay!

**Review by bibilujan — APPROVED** [review]

**Review by pruzanov — APPROVED** [review]

**Comment by pruzanov** [comment]
I approved, but it looks like the build is failing. It may be transient, need to check what Jenkins says

**Pull request merged**